### PR TITLE
Assume org support

### PR DESCRIPTION
**Pull request by zeritte**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,14 +1,24 @@
 # Changelog
-* Friday 20th May 2022
-v0.16.2
+
+- Friday \*\* June 2022
+  v0.16.2
+
+## Enhancements
+
+- We introduced a new function called `set_assume_org_id` for incoming super organization feature. By using this method, you can assume the id of a child organization as their super organization, and make your calls on behalf of them.
```
Comment (springcoil) [inline CHANGELOG.md:8]: I think here we need to explain what a superorganisation is.

File: README.md [hunk 2]
```diff
@@ -187,6 +187,21 @@ export apiaudio_key=<your-key>
...
+
+Set your child organization id to `None` to stop assuming an organization.
+
+```python
+import apiaudio
+
+apiaudio.set_assume_org_id('child_org_id')
+
+# Stop using
+apiaudio.set_assume_org_id(None)
```
Comment (springcoil) [inline README.md:202]: Why is None not a string?

Comment (springcoil) [reply README.md:202]: And why None - and not false?

**Review by springcoil — CHANGES_REQUESTED** [review]
I've a few questions

**Review by LindaKivine — APPROVED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (zeritte) [reply CHANGELOG.md:8]: agreed

File: README.md (hunk 2 above)
Comment (zeritte) [reply README.md:202]: simply thinking, i assume no org, so i am myself. or i assume an org.

**Review by guillem-211 — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (Sjhunt93) [reply README.md:202]: For syntax and usability we could have a reset_assume_org_id() ? 

its an extra function but it makes it very clear,

whereas set_assume_org_id(false) or set_assume_org_id("") actually implies you don't use a org at all

**Review by martinezpl — APPROVED** [review]
Nice and simple.

**Comment by springcoil** [comment]
@zeritte Do you want me to add in organisations?

**Pull request merged**